### DISCUSSION TITLE
Subtitle download

### DISCUSSION THREAD

**Discussion by mirakklys** [ilyhalight/voice-over-translation#330] [Ideas]
Would it be possible to add "Download subtitle" button? Now I can only see the overlay of the subtitles. Thank you!

**Comment by ilyhalight** [comment]
You can download the subtitles you need via the browser console
![image](https://github.com/ilyhalight/voice-over-translation/assets/62353659/90eeb34d-5fd2-4b9a-a52b-c913bb04da86)

**Reply by mirakklys** [reply]
Great! Thank you so much!:)
